### DISCUSSION TITLE
ORC efficiency above 200C correct?

### DISCUSSION THREAD

**Discussion by softwareengineerprogrammer** [NatLabRockies/GEOPHIRES-X#196] [Ideas]
Per user:

> I noticed, as in Geophires 2, that the subcritical ORC option can be selected at any temperature, eg 250 Deg C. Please can you check whether this is physically correct. And if so, if the very high heat to electricity conversion efficiencies are correct?
> The subcritical ORC question is pretty important. I want to be sure the software is not (greatly) overstating the heat to electricity conversion efficiency above 200 Deg C. In Beckers’ PhD thesis 2016 he drew a line to 200 Deg C for the validity of the ORC and then attached to that a line for double flash, which hardly increases at all with temperature. So I presumed that 200 Deg C was the limit for ORC and we would receive marginal additional efficiency benefits above 200 Deg C (from double flash)

---

Tracking issue: https://github.com/NREL/GEOPHIRES-X/issues/358

**Comment by softwareengineerprogrammer** [comment]
@kfbeckers can you weigh in on this, specifically the part about ORC efficiency above 200C?

**Comment by kfbeckers** [comment]
We only evaluated ORC efficiency for geothermal temperatures between 100 and 200C. For higher temperatures, GEOPHIRES would be extrapolating the correlations for an ORC which is likely only approximately correct. I would expect, yes, ORC efficiencies further increase beyond 200C. If you have your own data, e.g., from power plant simulations, you could overwrite the correlation value. Note that the efficiency GEOPHIRES used is based on the exergy ("utilization efficiency" or "second law efficiency" ) and not based on the heat production ("thermal efficiency" or "heat-to-power efficiency")

**Comment by softwareengineerprogrammer** [comment]
Per @kfbeckers: Results from IPSEpro Closed-Loop Geothermal Working Group may provide data to refine these calculations
